src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100000.mdx

```diff
@@ -0,0 +1,76 @@
+
+This release of the Python agent drops support for Python2.7, adds a Kafka server metric, capturing of memcache host and port, support for [kafka-python-ng](https://pypi.org/project/kafka-python-ng/), support for [uvicorn_worker](https://pypi.org/project/uvicorn-worker/), an environment variable for GC runtime metrics, and automatically detects the function signature in [LangChain](https://pypi.org/project/langchain/) `similarity_search`. It also fixes a bug in [graphql](https://pypi.org/project/graphql/) entity name detection, an issue in [LangChain](https://pypi.org/project/langchain/) where a thread is started and the transaction context is lost, parsing of the request and response when running [OpenAI](https://pypi.org/project/openai/) via [LangChain](https://pypi.org/project/langchain/), [pyscopg](https://pypi.org/project/psycopg/) v3 API incompatibilities, and instruments a new path to the [OpenAI](https://pypi.org/project/openai/) chat class.
+ 
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+## New features
+
+* Add Kafka server metric
+
+  * Adds a metric to capture the host and port information for Kafka entities on both consumers and producers. This metric allows the UI to link AWS information with AWS MSK entities in the service map.
+
+* Add capturing of memcache host and port
+
+  * Adds capturing of host and port info on memcache libraries; bmemcached, aiomcache, and pymemcache. This allows the UI to link AWS information with AWS memcache entities in the service map.
+
+* Add support for [kafka-python-ng](https://pypi.org/project/kafka-python-ng/)
+
+  * [kafka-python](https://pypi.org/project/kafka-python/) has been released under a new name: [kafka-python-ng](https://pypi.org/project/kafka-python-ng/). This continues to support kafka-python under the new package name.
+
+* Add support for [uvicorn_worker](https://pypi.org/project/uvicorn-worker/)
+
+  * `uvicorn.workers` has been moved to a separate package called `uvicorn_worker`. This checks for both names of the module when reporting dispatcher information.
+
+* Automatically detect function signature in [LangChain](https://pypi.org/project/langchain/) `similarity_search`
+
+  * Automatically detect the function signature when wrapping `similarity_search`.
+
+* Add environment variable for GC runtime metrics
+
+  * GC runtime metircs can now be enabled/disabled via the following environment variable: NEW_RELIC_GC_RUNTIME_METRICS_ENABLED. 
+
+## Deprecations
+
+* Remove Python2.7 Support
+
+  * Removes compatibility with Python2.7. Also removes instrumentations that were only supported on Python2.7.
+
+## Bug fixes
+
+* Fix bug in graphql entity name detection
+
+  * Previously, the graphql channel entity name would be missing the first couple characters. This bug has been fixed.
```

Comment:
Is this the PR this is referencing: https://github.com/newrelic/newrelic-python-agent/pull/1206--if so, this is gRPC and we should change that in the summary, metadata, and this section

Assuming this is for gRPC, let's change the sentence to something like this:
Previously, the gRPC channel entity name was missing the first character(s) in specific cases. This bug has been fixed.

Reply:
OMG Yes that is the PR and I'm special.